api/stats/internal.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
```

Comment:
Can we rename this proto file to something more specific to its content? E.g. `statistic.proto` or similar? This is to avoid attracting pretty much any internal messages and keep the files focused.

We can create a subdirectory `internal/` for internal protos and we probably don't want that to live under `api/`, since that implies external use. We could create a directory called `internal_proto` or similar somewhere else.

Also if these proto are only meant for internal use, this could be a good use case to introduce a `nighthawk::internal` namespace. Up to you whether you think this is necessary. It would remove the need of prepending all their names with "Internal".

Reply:
Pushed 8ce99fc9 to restructure accordingly